sway-lib-std/src/vec.sw

```diff
@@ -208,4 +208,22 @@ impl<T> Vec<T> {
         self.len -= 1;
         Option::Some(read(self.buf.ptr() + self.len * size_of::<T>()))
     }
+
+    /// Swaps two elements
+    /// a - The index of the first element
+    /// b - The index of the second element
+    /// Panics if `a >= self.len || b >= self.len` .
```

Comment:
Taking from https://doc.rust-lang.org/src/core/slice/mod.rs.html#593-604

```suggestion
    /// Swaps two elements.
    ///
    /// # Arguments
    ///
    /// * a - The index of the first element
    /// * b - The index of the second element
    ///
    /// # Reverts
    ///
    /// Reverts if `a` or `b` are out of bounds.
```

Reply:
Guess I'll start bringing this up outside of sway-apps too